runtime/doc/api.txt

```diff
@@ -3378,6 +3388,40 @@ nvim_tabpage_list_wins({tabpage})                   *nvim_tabpage_list_wins()*
     Return: ~
         List of windows in `tabpage`
 
+                                                   *nvim_tabpage_set_layout()*
+nvim_tabpage_set_layout({tabpage}, {layout})
```

Comment:
Reviewing this, I think it wouldn't be clumsy if we renamed this to `nvim_tabpage_set`, and returned a dict which puts the layout in a `layout` field. That allows us to add other stuff in the future.

Also need a `opts` parameter for future extensibility.

```suggestion
nvim_tabpage_get({tabpage}, {config})
```